src/app/shared/services/api.service.ts

```diff
@@ -1,35 +1,131 @@
+import {
+  PersonImagesResult,
+  PersonImagesResultApi,
+} from 'src/app/actor-page/models/person-images-result';
+import { Person, PersonApi } from 'src/app/actor-page/models/person.model';
+import {
+  PersonCredits,
+  PersonCreditsApi,
+} from 'src/app/actor-page/models/person-credits.model';
+import { Observable, map } from 'rxjs';
 import { SearchResultAPI } from 'src/app/main/models/search-result.model';
-import { map, Observable } from 'rxjs';
-import { Genres } from 'src/app/main/models/genres.model';
 import { MoviesSearchResult } from 'src/app/main/models/search-result.model';
-import { Api } from 'src/app/main/enums/api.enum';
+import { Genres } from 'src/app/main/models/genres.model';
+import { Api, PeopleApi, TabPath } from 'src/app/main/enums/api.enum';
+import {
+  MovieFullInfo,
+  MovieFullInfoApi,
+} from 'src/app/movie-page/models/movie-full-info.model';
+import { MovieCredits } from 'src/app/movie-page/models/movie-credits.model';
+import {
+  MovieImages,
+  MovieImagesApi,
+} from 'src/app/movie-page/models/movie-images.model';
+import { MovieApi } from 'src/app/main/enums/api.enum';
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { convertApiResultToResult } from 'src/app/main/models/converters/convertApiResultToResult';
+import { convertFulMovieInfoToMovie } from 'src/app/movie-page/models/converters/convertFullMovieInfoToMovie';
+import { convertImagesApiToImages } from 'src/app/movie-page/models/converters/convertImagesApiToImages';
+import { convertPersonCreditsToMovie } from 'src/app/actor-page/models/converters/convertPersonCreditsToMovie';
```

Comment:
in order not to do a bunch of similar imports, I recommend using index.ts to group files